addons/containerd/1.3.7/install.sh

```diff
@@ -3,17 +3,19 @@ CONTAINERD_NEEDS_RESTART=0
 function containerd_install() {
     local src="$DIR/addons/containerd/$CONTAINERD_VERSION"
 
-    logStep "Installing Containerd host packages"
```

Comment:
can we keep the logstep?

Reply:
Or that got moved inside the function

Reply:
Ya, it got moved inside the function

Reply:
https://github.com/replicatedhq/kURL/pull/1689/files#diff-eac01f76ffb85a6836a92bcebb5ebcce72df95758f7ca3c6c6158d8337c26f7fR88